contracts/WamplLoanRouter.sol

```diff
@@ -0,0 +1,111 @@
+pragma solidity ^0.8.3;
+
+import "./interfaces/ILoanRouter.sol";
+import "./interfaces/IBondController.sol";
+import "./interfaces/ITranche.sol";
+import "./interfaces/IButtonWrapper.sol";
+import "./interfaces/IWAMPL.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./interfaces/IWamplLoanRouter.sol";
+import "./test/WAMPL.sol";
+
+/**
+ * @dev Wampl Loan Router built on top of a LoanRouter of your choosing
+ * to allow loans to be created with raw ampl instead of WAMPL
+ */
+contract WamplLoanRouter is IWamplLoanRouter {
```

Comment:
Should test, but worth considering just having this inherit `LoanRouter` and having it as a fully functional separate instance instead of a composition. Probably saves quite a bit of gas from the external calls

Reply:
@marktoda I'm not sure how much sense this makes since it will never make use of the `borrow` or `borrowMax` function. It only uses the `wrapAndBorrow` or `wrapAndBorrowMax` functions. No need to support those extra functions.

Reply:
If the user came in with buttonWAMPL they could use `borrow` and `borrowMax` right?

Reply:
Yeah, they're still useful.. In any case I think we should do this - saves a lot of runtime gas at the cost of a little deployment gas (a tradeoff we should always be willing to make for contracts that are deploy-once use-many IMO)